.changeset/thick-bears-buy.md

```diff
@@ -0,0 +1,8 @@
+---
+"docs": patch
```

Comment:
`docs` shouldn't be included as part of this changeset as we only want to bump public packages.

Reply:
I add what the default changeset provides 😄  (we should adjust the settings)